source/community/release_notes/sumac/aspects_comp_dashboard.rst

```diff
@@ -1,11 +1,38 @@
 Aspects Course Comparison Dashboard
 ###################################
 
-The Product and Engineering Teams from eduNEXT and Axim Collaborative are
-excited to introduce a new Aspects Dashboard for course administrators with the
-upcoming Sumac Release! The new Course Comparison Dashboard will allow users to
-easily compare courses and runs by enrollment, number of active users,
-high-level performance and video engagement metrics. We've also made
-improvements to existing course-level dashboards to make them easier to use.
-
-The Open edX Sumac release comes out in December 2024.
\ No newline at end of file
+The Product and Engineering Teams from eduNEXT and Axim Collaborative are excited to introduce the `Aspects Course Comparison Dashboards </en/latest/documentors/references/quick_reference_rst.html>`_ for course administrators! We’ve also made improvements to existing course-level dashboards to make them easier to use. 
+
+Operators can learn more about enabling or updating Aspects on their installations by reviewing the Operator Release Notes.
+
+Compare courses and course runs with Aspects
+********************************************
+
+The new Course Comparison Dashboard allows users to easily compare courses and/or course runs by enrollment, number of active users, high-level performance and video engagement information. We’ve included filters and sorting functionality to help users make comparisons between courses. 
+
+In this new dashboard, users can view metrics for all runs of a course combined or broken out by each individual run by switching between two tabs at the top of the dashboard. Those who want to compare all runs of a course at a glance can easily do so by filtering the dashboard by course and viewing the Run Metrics tab.
+
+.. image:: /_images/sumac-aspects-run-metrics-1
+
+Spot high and low enrollment and activity
+=========================================
+
+The Course Comparison Dashboard makes it easy to identify courses with the highest and lowest levels of enrollment and activity with tables that allow users to sort courses or runs by current enrollment or active learner count. 
+
+.. image:: /_images/sumac-aspects-course-info-2
```

Comment:
You have to put `.png` at the end of these filenames! (full file names, not just title). Also the path to the folder is wrong.

```suggestion
.. image:: /_images/release_notes/sumac/sumac-aspects-course-info-2.png
```

Reply:
Be sure to fix all the filenames in this doc please :)